crates/module_system/ixc/examples/migrations.rs

```diff
@@ -139,45 +139,45 @@ mod tests {
         let mut bob = test_app.new_client_context().unwrap();
         let foo = create_account::<Handler1>(&mut bob, Handler1Create {}).unwrap();
         assert_eq!(
-            get_handler_id(&bob, foo.account_id()).unwrap(),
+            get_handler_id(&bob, foo.target_account()).unwrap(),
```

Comment:
target account is the account id of the target account?

Reply:
Yes, basically the account the client is sending messages to. I think that's clearer than what it was before